plugin.video.arloview/resources/lib/Arlo.py

```diff
@@ -0,0 +1,1085 @@
+# #
```

Comment:
A script.module add-on is a python module packaged seperately. Then you just import it (like script.module.requests). The advantage is that it can be seperately updated and if multiple add-ons use it, they don't have to ship it themself.

You can look at the layout of one at https://github.com/xbmc/repo-scripts/tree/leia/script.module.requests. If you still need some help, I'll assist.

Reply:
the Arlo.py is a module written by a different author.  He is aware that I am building a Kodi plugin with his module.  How do I create addon.xml and the other addon information (website, source email)?  
As this is a specific module related to this plugin, I put it in the lib directory.

Reply:
I'm aware that it is written by a different author. As long as you comply with the licence of that module all is fine.
You can look at https://github.com/xbmc/repo-scripts/compare/leia...Rechi:script.module.arlo to see that module packaged. You can just pick that for testing and submitting if it works.